tessellation/src/path_stroke.rs

```diff
@@ -660,6 +660,44 @@ impl<'l, Output: 'l + GeometryBuilder<Vertex>> StrokeBuilder<'l, Output> {
                 }
             }
 
+            LineJoin::Bevel => {
+            
+                // First we take our initial vertex for compute our triangle
+                let current_vertex = add_vertex!(
+                    self,
+                    Vertex {
+                        position: self.current,
+                        normal: normal,
+                        advancement: self.length,
+                        side: front_side,
+                        }
+                );
+
+                // Compute 2 normal on the right and left side  
+                let normal = if front_side == Side::Right {
+                    vec2(a_line.y, -a_line.x).normalize() * 0.5
+                } else {
+                    vec2(-a_line.y, a_line.x).normalize() * 0.5
+                };
+
+                // Now we can add vextex at the end of our normals
+                let mut last_vertex = add_vertex!(
+                    self,
+                    Vertex {
+                        position: self.current,
+                        normal: normal,
+                        advancement: self.length,
+                        side: front_side,
+                        }
+                );
+                let back_vertex = last_vertex;
```

Comment:
I think that `back_vertex` here should be the one already created, instead of using `last_vertex` (otherwise it means `last_vertex` and `back_vertex` are the same vertex and the resulting triangle is infinitely thin).